rclcpp/include/rclcpp/node.hpp

```diff
@@ -235,6 +235,12 @@ class Node : public std::enable_shared_from_this<Node>
   rcl_interfaces::msg::SetParametersResult
   set_parameters_atomically(const std::vector<rclcpp::parameter::ParameterVariant> & parameters);
 
+  template<typename ParameterT>
+  void
+  set_parameter_if_not_set(
+    const std::string & name,
+    const ParameterT & default_value);
```

Comment:
NIt: "default_value" -> "value" or "new_value", default doesn't really carry meaning in this new approach. Same below